include/mudObjects/creatures.hpp

```diff
@@ -686,6 +698,9 @@ class Creature: public virtual MudObject, public Streamable, public Container, p
     void fixLts();
     void doDispelMagic(int num=-1);
     bool changeSize(int oldStrength, int newStrength, bool enlarge);
+    bool checkResistEnchantments(const std::shared_ptr<Creature>& caster, const std::string spell, bool print=false);
+    bool isMagicallyHeld(bool print=false) const;
```

Comment:
Where is the definition of this function? Not in this PR?

Reply:
It's part of the hold spell stuff i put in that's in playtesting. Also has an effect on scare and fear spell.

Reply:
The definition of what the function does is also above the function in comments.